dataactvalidator/config/sqlrules/c23_award_financial_1.sql

```diff
@@ -1,24 +1,31 @@
 -- For each unique PIID for procurement in File C where ParentAwardId is null, the sum of each
 -- TransactionObligatedAmount submitted in the reporting period should match (in inverse) the sum of the
 -- FederalActionObligation amounts reported in D1 for the same timeframe, regardless of modifications.
+WITH award_financial_b23_1_{0} AS
+    (SELECT submission_id,
+    	piid,
+    	transaction_obligated_amou,
+    	parent_award_id,
+    	allocation_transfer_agency
+    FROM award_financial
+    WHERE submission_id = {0})
 SELECT
 	NULL as row_number,
 	af.piid,
 	(SELECT COALESCE(SUM(sub_af.transaction_obligated_amou::numeric),0) AS transaction_sum
-		FROM award_financial as sub_af WHERE submission_id = {0} AND sub_af.piid = af.piid AND COALESCE(sub_af.parent_award_id,'') = '') AS transaction_obligated_amou_sum,
+		FROM award_financial_b23_1_{0} as sub_af WHERE submission_id = {0} AND sub_af.piid = af.piid AND COALESCE(sub_af.parent_award_id,'') = '') AS transaction_obligated_amou_sum,
```

Comment:
Don't think we need `submission_id` in the WHERE clause now that we're selecting from the temp table